lizmap/modules/lizmap/lib/Server/Server.php

```diff
@@ -249,6 +249,10 @@ private function getLizmapMetadata()
             $data['hosting'] = \jApp::config()->lizmap['hosting'];
         }
 
+        if (\jApp::isModuleEnabled('webdav') && class_exists('\\LizmapDAV\\Server')) {
```

Comment:
```suggestion
        if (\jAcl2::check('lizmap.admin.server.information.view') && \jApp::isModuleEnabled('webdav') && class_exists('\\LizmapDAV\\Server')) {
```

Otherwise, it's visible without being connected.

Reply:
@laurentj I agree with @Gustry suggestion